compare258.c

```diff
@@ -155,7 +155,7 @@ static inline int32_t compare256_unaligned_64_static(const unsigned char *src0,
 
         if (diff) {
             uint64_t match_byte = __builtin_ctzll(diff) / 8;
-            return (int32_t)(len + match_byte);
+            return (uint32_t)(len + match_byte);
```

Comment:
```len``` and ```match_byte``` are both uint32_t, so the cast is superfluous.

Reply:
Fixed